ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/search/presentation/FindResultViewImpl.java

```diff
@@ -48,6 +48,9 @@ public FindResultViewImpl(
     NodeLoader loader = new NodeLoader(Collections.<NodeInterceptor>emptySet());
     tree = new Tree(nodeStorage, loader);
 
+    tree.ensureDebugId("resultSearchTree");
```

Comment:
IMHO it's better to explain proposal of setup debug id here to prevent occasional removal in future.